### PR TITLE
CASH-551 - Sort loans in lend/filter

### PR DESCRIPTION
**Pull request by foochris**

### CODE REVIEW

File: src/pages/Lend/Filter/FilterSectionSort.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<template>
+	<filter-menu-section :title="title" :open="open" :result-count="resultCount">
+		<ais-sort-by :items="defaultSortIndices">
```
Comment (mcstover) [inline src/pages/Lend/Filter/FilterSectionSort.vue]: Please pass defaultSortIndices down into this component. This so we don't have to include configure or the mixin here.

Alternatively, you could pull the fundraisingIndices + defaultSortIndices() out of algolia-config-mixin into a 2nd algolia-sort-indices-mixin.

Comment (foochris) [reply src/pages/Lend/Filter/FilterSectionSort.vue]: @mcstover PR updated.

File: src/pages/Lend/Filter/FilterSectionSort.vue [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import FilterMenuSection from '@/pages/Lend/Filter/FilterMenuSection';
+
+export default {
+	components: {
+		AisConfigure,
+		AisSortBy,
+		FilterMenuSection,
+	},
+	filters: {
+		capitalize(value) {
```
Comment (foochris) [inline src/pages/Lend/Filter/FilterSectionSort.vue]: @mcstover Should we move this globally? There may be another use for it in `SelectedRefinements.vue` as the label 'female / male' should be upper case.

File: src/pages/Lend/Filter/LendFilterMenu.vue [hunk 3]
```diff
@@ -78,11 +80,13 @@
 
 <script>
 
+import algoliaConfig from '@/plugins/algolia-config-mixin';
```
Comment (mcstover) [inline src/pages/Lend/Filter/LendFilterMenu.vue]: @foochris this is already included in the page. please pass down from there.

Comment (foochris) [reply src/pages/Lend/Filter/LendFilterMenu.vue]: @mcstover PR updated.

**Pull request merged**